### ISSUE TITLE
vfmt replaces inline /* */ with single line comment

### ISSUE THREAD

**Issue by serkonda7** [vlang/v#6201]
**V version:** V 0.1.29 3e5871f
**OS:** Manjaro

**What did you do?**
Run fmt on this:
```v
fn main() {
	arr := ['abc', /* '123', */ 'def',]
}
```

**What did you expect to see?**
No changes
 
**What did you see instead?**
```v
fn main() {
        arr := ['abc', // '123', 'def']
}
```

Running the fmt'ed code will give the error:
```
invalid_vfmt.v:3:1: error: expr(): bad token `}`
    1 | fn main() {
    2 |     arr := ['abc', // '123', 'def']
    3 | }
      | ^
```

**Comment by Larpon** [comment]
Can confirm with `fa3e952`

**Comment by serkonda7** [comment]
Fixed by f42a44abec6decced1e77021723fe81121da1acf
